### ISSUE TITLE
Component Width Resizer Widget

### ISSUE THREAD

**Issue by RobLoach** [kss-node/kss-node#504]
allow resizing the widget on screen so that you don't have to resize the window.
![screenshot at 2018-08-27 15-24-11](https://user-images.githubusercontent.com/25086/44681015-50b2ec00-aa0d-11e8-8855-749b9a7d1df9.png)

**Comment by ryuran** [comment]
@RobLoach by “widget” you mean demo rendering area (content on “Example” block on your screenshot) ?

**Comment by RobLoach** [comment]
Yes, that :wink:

**Comment by ryuran** [comment]
I’m ok with that.

Some other things can be improve for demo area, it could be:
 - resizable
 - isolated from KSS css and js
 - responding to mediaquery

I have only one solution to that: iframe

This script seems to do the job : https://github.com/edenspiekermann/iframify
But too many iframes in a page is greedy in performance.

**Comment by kyleoliveiro** [comment]
You could use a custom element with Shadow DOM, as an alternative to iframes.

**Comment by charlieartist** [comment]
Please correct me if this is totally off-base, but are you looking for functionality similar to what [Shopify Polaris](https://polaris.shopify.com/components/forms/autocomplete#navigation) uses in their demos?

**Comment by RobLoach** [comment]
That'd be cool!

**Comment by charlieartist** [comment]
@RobLoach Yes—it would be a very nice IXD/UX addition to the stack. Right now, I've done a fair bit of customization to a KSS-Node install, but mostly through CSS with some minor Twig. 

Elements such as this add real polish to a guide, besides being an interactive demo. And, it's another affordance to invite visitors to test responsiveness.
